### PR TITLE
Formatting views.py, moving NLP code to services_nlp.py

### PR DESCRIPTION
**Pull request by simon-tarr**
I've formatted views.py with isort and PEP8 (using Black).

I've started moving some of the NLP code to it's own services file. I will move more later, once views.py has been merged into master.

### CODE REVIEW

File: api/mapping/services_nlp.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    url = "https://ccnett2.cognitiveservices.azure.com/text/analytics/v3.1-preview.3/entities/health/jobs?stringIndexType=TextElements_v8"
+    headers = {
+        "Ocp-Apim-Subscription-Key": os.environ.get("NLP_API_KEY"),
+        "Content-Type": "application/json; utf-8",
+    }
+
+    # Create payload, POST to the NLP servoce
+    payload = json.dumps(chunk)
+    response = requests.post(url, headers=headers, data=payload)
+    print(response.status_code, response.reason)
```
Comment (py5gol) [inline api/mapping/services_nlp.py:41]: Remove or using logging please.

File: api/mapping/services_nlp.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    post_response_url = response.headers["operation-location"]
+    time.sleep(3)
+
+    print("PROCESSING JOB >>>", post_response_url)
```
Comment (py5gol) [inline api/mapping/services_nlp.py:45]: Remove.

File: api/mapping/services_nlp.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+    # GET the response
+    req = requests.get(post_response_url, headers=headers)
+    job = req.json()
+
+    # Loop to wait for the job to finish running
+    get_response = []
+    while job["status"] != "succeeded":
+        print(job["status"])
```
Comment (py5gol) [inline api/mapping/services_nlp.py:54]: Remove prints.

File: api/mapping/views.py [hunk 4]
```diff
@@ -32,42 +31,63 @@
...
+    OmopField,
+    OmopTable,
+    ScanReport,
+    ScanReportAssertion,
+    ScanReportField,
+    ScanReportTable,
+    ScanReportValue,
+    StructuralMappingRule,
+)
 from .services import process_scan_report, run_usagi
```
Comment (py5gol) [inline api/mapping/views.py:61]: Do we need run_usagi?

**Review by py5gol — CHANGES_REQUESTED** [review]

File: api/mapping/views.py (hunk 4 above)
Comment (simon-tarr) [reply api/mapping/views.py:61]: Good question. Depends if you think we'll want to use the Usagi code in the future. I'd be happy to delete it and never see it again :D

**Review by py5gol — APPROVED** [review]

**Pull request merged**